docs/static/monitoring/monitoring-mb.asciidoc

```diff
@@ -36,6 +36,20 @@ Remove the `#` at the beginning of the line to enable the setting.
 
 --
 
+[float]
+[[define-cluster__uuid]]
+==== Define `cluster_uuid`
+To bind the metrics of {ls} to a specifi cluster, define the `monitoring.cluster_uuid` 
+in configuration file (logstash.yml):
+
+
+[source,yaml]
+----------------------------------
+monitoring.cluster_uuid: PRODUCTION_ES_CLUSTER_UUID
+----------------------------------
+
+--
```

Comment:
```suggestion
```

Reply:
This block tag must be removed before we can pass docs-ci and get a clean build.  (The previous section uses delimiting blocks because of tagged regions. Not necessary or permitted here.)